phy/mod_blom_init.F90

```diff
@@ -150,7 +151,9 @@ subroutine blom_init()
     ! Initialize shortwave radiation absorption
     ! ------------------------------------------------------------------
 
-    call iniswa
+    if (.not. use_stream_swa) then
+       call iniswa
+    end if
```

Comment:
I fear there has been a misunderstanding about the HAMOCC swa field? For the bromoform-option there has been a swa-file, which contains a pre-industrial swa-climatology  (because bromoform production is parameterized using deviations from the swa rather than absolute values). This `iniswa` deals with how shortwave absorption is handled in BLOM and need to be called always.

The "normal" short-wave radiation field as far as I am aware *always* comes from CAM or DATM and is never read through a stream by BLOM. 

The reading of the swa-climatology is not done consistently with the other inputs (the bromoform code has not been used a lot). The climatology file is read in in `homocc_init` and the just used inside hamocc.

Reply:
@JorgSchwinger - thanks for clarifying. I should have seen this.